README.md

```diff
@@ -1,6 +1,30 @@
-# 10x
+# 10x website Problem Statement and Goals
+
+## Macro
+
+**Opportunity:** The U.S. Government employs many capable public servants who observe problems with government technology and are ideally positioned to pursue solutions to these problems. 
```

Comment:
Only thing – `government` should not be capitalized, per the 18F content guide. I'm going to go ahead and merge and we can do a quick follow up fix.